app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show
+    params[:users_id]
+  end
+end
```

Comment:
- [ **OPTIONAL** ] **In README.md file:** I think it would be better if you update your README by adding some important missing sections/sub-secctions(e.g.: `## Getting Started`, `### Setup`, `###Preriquisites`,...) and even update some content of some existed sections(e.g.: project description, `## Authors section`,...) to make your documentation more professional. You can use **[this template](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)** to help you complete all the related sections of your project.
e.g.:  @usernames_on_each_social_media
![Screenshot from 2022-11-08 03-24-06](https://user-images.githubusercontent.com/83042742/200452430-9a752ced-3c66-4286-ac13-4db88a5dd699.png)